frontend/server/minio-helper.ts

```diff
@@ -34,48 +37,155 @@ export interface MinioClientOptionsWithOptionalSecrets extends Partial<MinioClie
 }
 
 /**
- * Create minio client with aws instance profile credentials if needed.
+ * Create minio client for s3 compatible storage
+ *
+ * If providerInfoString is available, use these over defaultConfigs.
+ *
+ * If providerInfo is not provided or, if credentials are sourced fromEnv,
+ * then, if using aws s3 (via provider chain or instance profile), create a
+ * minio client backed by aws s3 client.
```

Comment:
Minio is the default deployment approach, Can we make sure this continues to be the case?

Reply:
@zijianjoy yes, the above description describes the aws case, if aws is not specified, we default to minio when using default deployment manifests (since that is what is configured in the kfp ui environment)